controller/search_blackbox_test.go

```diff
@@ -920,4 +920,49 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 		compareWithGolden(t, filepath.Join(s.testDir, "show", "non_existing_key.error.golden.json"), jerrs)
 		compareWithGolden(t, filepath.Join(s.testDir, "show", "non_existing_key.headers.golden.json"), res.Header())
 	})
+	s.T().Run("assignee=null after", func(t *testing.T) {
+		filter := fmt.Sprintf(`
+					{"$AND": [
+						{"assignee":null}
+					]}`,
+		)
+		_, result := test.ShowSearchOK(s.T(), nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
+		require.Empty(s.T(), result.Data)
+		assert.Len(s.T(), result.Data, 0) //resolved bugs
```

Comment:
please remove or update the comment `resolved bugs`

Reply:
Fixed in 17f24ffbdbfd1a28ea5ee5deed34389a6a9ad015